vivo-project/Vitro
# [VIVO-1690] - Remove trailing whitespace throughout project

**[JIRA Issue](https://jira.duraspace.org/browse/VIVO-1690)**: (https://jira.duraspace.org/browse/VIVO-1690)

* Other Relevant Links (Mailing list discussion, related pull requests, etc.)
https://softwareengineering.stackexchange.com/questions/121555/why-is-trailing-whitespace-a-big-deal
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

# What does this pull request do?
There is a lot of extraneous trailing whitespace throughout the VIVO codebase and I don't like it. This removes the extra whitespace and adds trailing newlines in files for max *nix compatibility.

# What's new?
Hopefully nothing, just a lot less whitespace. 

# How should this be tested?
There should be no functional changes to the application, just code style. You can confirm there are only whitespace changes in GitHub by appending &w=1 to the file diff page URL: https://github.com/vivo-project/Vitro/pull/117/files?w=1

# Additional Notes:
This is a situation where an existing and robust testing suite would be nice.

# Interested parties
@awoods
